src/test/scala/org/dia/tensors/perf/MainMemoryNd4JPerformanceTest.scala

```diff
@@ -17,33 +17,28 @@
  */
 package org.dia.tensors.perf
 
-
 import org.nd4j.linalg.factory.Nd4j
 import org.nd4s.Implicits._
-import org.scalatest.{FunSuite, Ignore}
+import org.scalatest.{ FunSuite, Ignore }
+
 /**
- * The Nd4j Performance Tests
- * Created by rahulsp on 7/7/15.
+ * Nd4j Performance Tests
+ * @author Created by rahulsp on 7/7/15.
```

Comment:
side note to us - we should probably remove all \@author tags from the code @legendOfZelda - thoughts? maybe a new PR/issue.